iso/config.go

```diff
@@ -43,6 +44,9 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	}
 	errs = packer.MultiErrorAppend(errs, c.ShutdownConfig.Prepare()...)
 	errs = packer.MultiErrorAppend(errs, c.CreateConfig.Prepare()...)
+	if matched, _ := regexp.MatchString(`^vmx-\d+$`, c.Version); !matched && c.Version != "" {
```

Comment:
Do we need such prefixes? New users will have troubles with understanding.
What if the parameter is just an integer?

Reply:
@mkuzmin the virtual machine hardware version must in the format `vmx-10` as an a string. The `vmx-` prefix is required. This is the format that the vCenter API requires.

I will change the code so that end users will specify an integer and we can do the string munging internal. I should have the code changes done shortly.

Reply:
Fixed